packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -35,7 +36,13 @@ typedef void GestureScaleStartCallback(Point focalPoint);
 typedef void GestureScaleUpdateCallback(double scale, Point focalPoint);
 
 /// Signature for when the pointers are no longer in contact with the screen.
-typedef void GestureScaleEndCallback();
+typedef void GestureScaleEndCallback(Velocity flingVelocity);
```

Comment:
I wonder if these should take details objects.  That's the pattern we've adopted for the other gestures.